### PR TITLE
Asynchronous API

### PR DESCRIPTION
**Pull request by datkat21**
Using async along with fetch is a better approach here.

```js
let x = require("./index");

x.glossary().then((r) => {
  console.log(r);
});
```

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,55 +1,55 @@
 module.exports = {
-  alerts: function alerts() {fetch(`https://api.weather.gov/alerts`).then(res => {return res.json()}).then(data => {console.log(data);}).catch(error => {console.log(error);});},
```
Comment (bramleyas) [inline index.js:2]: Really had to delete and replace all the lines? lmao

File: index.js [hunk 2]
```diff
@@ -1,55 +1,55 @@
...
+  zonesTypeZoneidForecast: function zonesTypeZoneidForecast(type, zoneId) {return new Promise(function (resolve, reject) { fetch(`https://api.weather.gov/zones/${type}/${zoneId}/forecast`).then(res => {return res.json()}).then(data => {resolve(data);}).catch(error => {reject(error);});})},
+  zonesForecastZoneidObservations: function zonesForecastZoneidObservations(zoneId) {return new Promise(function (resolve, reject) { fetch(`https://api.weather.gov/zones/forecast/${zoneId}/observations`).then(res => {return res.json()}).then(data => {resolve(data);}).catch(error => {reject(error);});})},
+  zonesForecastZoneidStations: function zonesForecastZoneidStations(zoneId) {return new Promise(function (resolve, reject) { fetch(`https://api.weather.gov/zones/forecast/${zoneId}/stations`).then(res => {return res.json()}).then(data => {resolve(data);}).catch(error => {reject(error);});})},
   helloWorld: function helloWorld() {console.log("Hello World")}
-};
```
Comment (bramleyas) [inline index.js:55]: you're taking the piss

**Review by bramleyas — APPROVED** [review]
Tested, works beautifully, makes the package actually usable. I'll update the documentation

File: index.js (hunk 1 above)
Comment (datkat21) [reply index.js:2]: I used a find and replace, maybe that messed with diff

File: index.js (hunk 2 above)
Comment (datkat21) [reply index.js:55]: i think my copy and paste omitted the newline

**Pull request merged**